bundles/org.openhab.binding.ahawastecollection/NOTICE

```diff
@@ -0,0 +1,13 @@
+This content is produced and maintained by the openHAB project.
+
+* Project home: https://www.openhab.org
+
+== Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License 2.0 which is available at
+https://www.eclipse.org/legal/epl-2.0/.
+
+== Source Code
+
+https://github.com/openhab/openhab-addons
```

Comment:
You need to add the jsoup license like here https://github.com/openhab/openhab-addons/blob/e24d43aecbcfc5af551c8e9087d9be6992e5c69a/bundles/org.openhab.binding.linky/NOTICE